templates/sms.1.html

```diff
@@ -0,0 +1 @@
+sms.1.txt
```

Comment:
The general template code assumes HTML versions of every template. Obviously that makes no sense for SMS, so I made it a symlink, which is what this is. But it doesn't need to be a symlink, it could be an empty file, or anything really.

Or I could add some conditional logic to the template code that ignores HTML files for `sms.*` templates, but that felt a bit like unnecessary complexity when I thought about it before.